lib/xslt/tei_to_html/lib/formatting.xsl

```diff
@@ -61,83 +73,77 @@
-    </xsl:when>
-    <xsl:when test="($DD != '') and ($MM != '')">
-      <xsl:call-template name="lookUpMonth"><xsl:with-param name="numValue" select="$MM" /></xsl:call-template>, <xsl:value-of select="$YYYY" />
-    </xsl:when>
-    <xsl:when test="($YYYY != '')">
-      <xsl:value-of select="$YYYY" />
-    </xsl:when>
-    <xsl:otherwise>
-      N.D.
-    </xsl:otherwise>
-  </xsl:choose>
-</xsl:template>
-
-<xsl:template name="lookUpMonth">
-  <xsl:param name="numValue" />
-  <xsl:choose>
-    <xsl:when test="$numValue = '01'">January</xsl:when>
-    <xsl:when test="$numValue = '02'">February</xsl:when>
-    <xsl:when test="$numValue = '03'">March</xsl:when>
-    <xsl:when test="$numValue = '04'">April</xsl:when>
-    <xsl:when test="$numValue = '05'">May</xsl:when>
-    <xsl:when test="$numValue = '06'">June</xsl:when>
-    <xsl:when test="$numValue = '07'">July</xsl:when>
-    <xsl:when test="$numValue = '08'">August</xsl:when>
-    <xsl:when test="$numValue = '09'">September</xsl:when>
-    <xsl:when test="$numValue = '10'">October</xsl:when>
-    <xsl:when test="$numValue = '11'">November</xsl:when>
-    <xsl:when test="$numValue = '12'">December</xsl:when>
-    <xsl:otherwise></xsl:otherwise></xsl:choose>
-</xsl:template>
-
-<!-- ================================================ -->
-<!--                      DAMAGE                      -->
-<!-- ================================================ -->
-
-<xsl:template match="damage">
-  <span>
-    <xsl:attribute name="class">
-      <xsl:value-of select="@type"/>
-    </xsl:attribute>
-    <xsl:apply-templates/>
-    <xsl:text>[?]</xsl:text>
-  </span>
-</xsl:template>
+  <!-- ================================================ -->
+  <!--                      DATES                       -->
+  <!-- ================================================ -->
+
+  <xsl:template name="extractDate">
+    <xsl:param name="date"/>
+    <!--This template converts a date from format YYYY-MM-DD to mm D, YYYY (MM, MM-DD, optional)-->
+
+    <xsl:variable name="YYYY" select="substring($date, 1, 4)"/>
+    <xsl:variable name="MM" select="substring($date, 6, 2)"/>
+    <xsl:variable name="DD" select="substring($date, 9, 2)"/>
+
+    <xsl:choose>
+      <xsl:when test="($DD != '') and ($MM != '') and ($DD != '')">
+        <xsl:call-template name="lookUpMonth"><xsl:with-param name="numValue" select="$MM"
+          /></xsl:call-template><xsl:text> </xsl:text>
+        <xsl:number format="1" value="$DD"/>, <xsl:value-of select="$YYYY"/>
+      </xsl:when>
+      <xsl:when test="($YYYY != '') and ($MM != '')">
+        <xsl:call-template name="lookUpMonth"><xsl:with-param name="numValue" select="$MM"
+          /></xsl:call-template>, <xsl:value-of select="$YYYY"/>
+      </xsl:when>
+      <xsl:when test="($DD != '') and ($MM != '')">
+        <xsl:call-template name="lookUpMonth"><xsl:with-param name="numValue" select="$MM"
+          /></xsl:call-template>, <xsl:value-of select="$YYYY"/>
```

Comment:
I'm also confused about these changes, which seem to just be a return character right in the middle of a tag?  Maybe it's just displaying a little weirdly in github?